presto-docs/src/main/sphinx/sql/revoke.rst

```diff
@@ -9,7 +9,7 @@ Synopsis
 
     REVOKE [ GRANT OPTION FOR ]
     ( privilege [, ...] | ALL PRIVILEGES )
-    ON [ TABLE ] table_name FROM ( user | USER user | ROLE role )
+    ON ( table_name | TABLE table_name | SCHEMA schema_name ) FROM ( user | USER user | ROLE role )
 
 Description
```

Comment:
Mention the semantic of revoke privileges to table (in separate commit) and then for schema